src/hooks/use-i18n-data.tsx

```diff
@@ -22,6 +22,7 @@ export const I18nDataProvider = ( { children }: { children: React.ReactNode } )
 
 	const initI18n = useCallback( async ( localeKey: SupportedLocale ) => {
 		const newI18n = createI18n( getLocaleData( localeKey )?.messages );
+		defaultI18n.setLocaleData( getLocaleData( localeKey )?.messages );
```

Comment:
This will be incorporated in a separate PR https://github.com/Automattic/studio/pull/504, so we could remove it.

Reply:
My bad!, I've removed it! https://github.com/Automattic/studio/pull/505/commits/fdd67795a72c110814de4a5afa80b9ce370338a6